index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>revera</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/style.css">
+  
+</head>
+<body>
+  <!-- nav bar-->
+    <nav class="container">
+      
+      
+
+      <div class="navbar-header">
+                 <a class="navbar-brand" href="#"> REVERA</a>
+                 <p class="navbar-text navbar-center"><a href="#" class="navbar-link"> Home</a></p>
```

Comment:
You can "split" you navbar, for a better item alignment, what I would suggest you is to use an unordered list ( this is a common way to build nav bar) and to use the navbar-right class on the top of it  : 


```html 

 <ul class="nav navbar-nav navbar-right " id="nav ">
                        <li>
                            <a href="# " class="active">Home</a>
                        </li>
                        <li>
                            <a href="# ">About</a>
                        </li>
                        <li>
                            <a href="# ">Sample page</a>
                        </li>
                        <li>
                            <a href="# ">Portofolio</a>
                        </li>
                        <li>
                            <a href="# ">Blog</a>
                        </li>
                        <li class="dropdown">
                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown
                                <span class="caret"></span>
                            </a>
                            <ul class="dropdown-menu">
                                <li>
                                    <a href="#">Action</a>
                                </li>
                                <li>
                                    <a href="#">Another action</a>
                                </li>
                                <li>
                                    <a href="#">Something else here</a>
                                </li>

                            </ul>
                        </li>
                        <li>
                            <a href="# ">Contact</a>
                        </li>
                    </ul>

```